app/exporters/dhis2_exporter.rb

```diff
@@ -1,65 +1,68 @@
-class DHIS2Exporter
-  require "dhis2"
+class Dhis2Exporter
+  require 'dhis2'
+  attr_reader :facility_identifiers, :periods, :data_elements_map, :category_option_combo_ids
 
-  def self.export
-    new.export
-  end
-
-  def initialize
-    abort("DHIS2 export not enabled in Flipper") unless Flipper.enabled?(:dhis2_export)
+  def initialize(facility_identifiers:, periods:, data_elements_map:, category_option_combo_ids: [])
+    @facility_identifiers = facility_identifiers
+    @periods = periods
+    @data_elements_map = data_elements_map
+    @category_option_combo_ids = category_option_combo_ids
 
     Dhis2.configure do |config|
-      config.url = ENV.fetch("DHIS2_URL")
-      config.user = ENV.fetch("DHIS2_USERNAME")
-      config.password = ENV.fetch("DHIS2_PASSWORD")
-      config.version = ENV.fetch("DHIS2_VERSION")
+      config.url = ENV.fetch('DHIS2_URL')
```

Comment:
[nit] standardrb fails on CI. You'll need to setup Rubymine's linter to use standardrb I think, it uses rubocop by default.

Reply:
Can also run `standardrb --fix` locally to fix all these.